### ISSUE TITLE
Bold font is not working for IBM Plex Sans from Google Fonts

### ISSUE THREAD

**Issue by 0x1a1a1a** [jgraph/drawio#3907]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

If you are technical, you should reporting bugs along the lines of https://marker.io/blog/how-to-write-bug-report. If you are not technical, we will make allowances, please try to make an effort to understand the process.

**Describe the bug**
IBM Plex Sans is not working in bold version.

**To Reproduce**
1. Enter any text and select it
2. Go to Text panel, click Font dropdown
3. Click Custom, click Google Fonts radio button, enter `IBM Plex Sans`, click Apply
4. Click Bold style icon

**Expected behavior**
Text style should have changed to bold. Bold style icon should be in Active state

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 22.0.2

**Desktop (please complete the following information):**
 - OS: macOS 12.6.9
 - Browser Chrome
 - Browser Version 117.0.5938.149

**Smartphone (please complete the following information):**

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
-
yes
**Additional context**

**Comment by davidjgraph** [comment]
Seems to work ok in FF. In chrome you can see the difference if you zoom in, the difference between bold and normal is hard to see at 100%.

I'd say this is fully outside our control, unfortunately.

**Comment by ihorinter** [comment]
@davidjgraph Thank you for your reply! However, it looks like something is not right with how fonts are imported, managed and displayed. I’ve created a comparison image with a rendering of a bold font in Figma as a reference. Draw.io just adds an extrathin stroke around letters in Chrome, and also increases tracking in Safari

<img width="782" src="https://github.com/jgraph/drawio/assets/59264318/400ffafa-b286-495d-97bc-cc5f1cf4c512">
4

**Comment by alderg** [comment]
Try replace the fonts source in the cell style with the following:
`fontSource=https%3A%2F%2Ffonts.googleapis.com%2Fcss2%3Ffamily%3DRoboto%3Awght%40400%3B700`

<img width="482" alt="Screenshot 2023-10-05 at 08 22 22" src="https://github.com/jgraph/drawio/assets/921200/93d87fb4-5777-4d91-b1c9-d5a0544419e8">

**Comment by ihorinter** [comment]
@alderg Hi! How and why to do this?

**Comment by alderg** [comment]
It's a workaround for the bold font issue. Select the cell and select Edit Style (Ctrl+E).

**Comment by ihorinter** [comment]
@alderg Thank you! I don’t have `fontSource` in this window:
`rounded=1;whiteSpace=wrap;html=1;strokeWidth=3;spacingTop=0;spacing=0;strokeColor=#8C9EF8;perimeterSpacing=4;fillColor=#FAFBFF;fontColor=#000000;glass=0;shadow=0;labelBackgroundColor=none;spacingLeft=16;spacingBottom=0;spacingRight=16;verticalAlign=middle;`

* I am using IBM Plex Sans font, not Roboto
* I want to set bold text for a part of a text, not to the whole block
* The solution itself seems very complicated for everyday use

**Comment by alderg** [comment]
It's a workaround, not the final solution. We'll fix it propertly in one of the upcoming releases (and can't we an ETA). If there is no foundSource in the cell style then you're probably not using that font. You'd have to attach the diagram file for us to investigate further.

**Comment by ihorinter** [comment]
@alderg Great, thank you! I am not demanding any ETA for a great free software, just pointing to a (probably) bug 🙂
```
<mxfile host="app.diagrams.net" modified="2023-10-05T13:14:28.981Z" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36" etag="T1wvBnQhwONtRR1lU6xO" version="22.0.2" type="google">
  <diagram name="Page-1" id="KJehN0ePQQ7_5QWpz20F">
    <mxGraphModel grid="0" page="0" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0">
      <root>
        <mxCell id="0" />
        <mxCell id="1" parent="0" />
        <mxCell id="MzHH_R3-6Q4yn6_Wetj4-1" value="&lt;p style=&quot;line-height: 150%; font-size: 20px;&quot;&gt;&lt;font style=&quot;font-size: 20px;&quot; data-font-src=&quot;https://fonts.googleapis.com/css?family=IBM+Plex+Mono&quot;&gt;&lt;font face=&quot;IBM Plex Sans&quot; data-font-src=&quot;https://fonts.googleapis.com/css?family=IBM+Plex+Sans&quot; style=&quot;font-size: 20px;&quot;&gt;System displays&lt;br&gt;&lt;/font&gt;&lt;/font&gt;&lt;span style=&quot;font-family: &amp;quot;IBM Plex Sans&amp;quot;;&quot;&gt;&lt;b&gt;Login&lt;/b&gt;&lt;/span&gt;&lt;span style=&quot;background-color: initial; font-family: &amp;quot;IBM Plex Sans&amp;quot;;&quot;&gt; page&lt;/span&gt;&lt;/p&gt;" style="rounded=1;whiteSpace=wrap;html=1;strokeWidth=3;spacingTop=0;spacing=0;strokeColor=#8C9EF8;perimeterSpacing=4;fillColor=#FAFBFF;fontColor=#000000;glass=0;shadow=0;labelBackgroundColor=none;spacingLeft=16;spacingBottom=0;spacingRight=16;verticalAlign=middle;" vertex="1" parent="1">
          <mxGeometry x="-832.0000000000001" y="192" width="256" height="128" as="geometry" />
        </mxCell>
      </root>
    </mxGraphModel>
  </diagram>
</mxfile>
```

**Comment by davidjgraph** [comment]
Fix in 22.1.4.
